src/modules/app/containers/RouterFragment.js

```diff
@@ -2,26 +2,17 @@ import { Fragment } from 'redux-little-router'
 import React from 'react'
 import PropTypes from 'prop-types'
 import RouteConfig from '../RouteConfig'
-import { connect } from 'react-redux'
 import LivingFormPage from '../../../routes/living-form/containers/LivingFormPage'
-import LivingPage from '../../../routes/living/containers/LivingPage'
+import LivingPage from '../../../routes/categories/containers/CategoriesPage'
```

Comment:
To stay consistent we should import it as CategoriesPage then.

Reply:
Right, thanks